### PR TITLE
Website Update RF#1

### PR DESCRIPTION
**Pull request by heelflipp900**
Several changes across the website
- Responsive carousel and fixed aspect ratio
- Major updates to text all over the website
- Fixed pictures all around the website
- Removed firebase dependence, hardcoded text lists.

### CODE REVIEW

File: _includes/aboutContent.html [hunk 1]
```diff
@@ -1,50 +1,33 @@
...
-	<!-- ExtraC -->
 	<div class="panel panel-info">
   		<div class="panel-heading">Extracurricular</div>
   		<div class="panel-body">
-    		<div class="media">
-    			<div class="media-right">
-    				<img class="media-object member-image" src="/img/aboutUsSnowboarding.jpg"/>
-    			</div>
-    		</div>
+    				<img class="media-object" src="img/aboutUsSnowboarding.jpg" width=600px/>
```
Comment (ekamp) [inline _includes/aboutContent.html]: Dont set a static width here, this should be fluid.

File: _includes/indexBottomPanel.html [hunk 2]
```diff
@@ -3,9 +3,52 @@
   <div class="row">
     <div class="col-md-4">
       <div class="panel panel-info">
-        <div class="panel-heading"><h3>Summer Practice Schedule</h3></div>
+        <div class="panel-heading"><h3>Spring 2016 Practice Schedule</h3></div>
         <div class="panel-body">
-          <ul class="list-group" id="PracticeList"/>
+          <ul class="list-group"/>
+          <li class="list-group-item">
+            <b>Sunday: </b>
```
Comment (ekamp) [inline _includes/indexBottomPanel.html]: This info should be coming from firebase not hardcoded

File: _includes/indexBottomPanel.html [hunk 3]
```diff
@@ -13,7 +56,19 @@
       <div class="panel panel-danger">
         <div class="panel-heading"><h3 class="center-text">Upcoming Races</h3></div>
         <div class="panel-body">
-          <ul class="list-group" id="RaceList"/>
+          <ul class="list-group"/>
```
Comment (ekamp) [inline _includes/indexBottomPanel.html]: Should switch to firebase

**Comment by ekamp** [comment]
Comments to fix

File: _includes/aboutContent.html (hunk 1 above)
Comment (heelflipp900) [inline _includes/aboutContent.html]: @ekamp Yea this was more of a quick fix. This iteration is not really ready for all kinds of resolutions.

File: _includes/indexBottomPanel.html (hunk 2 above)
Comment (heelflipp900) [inline _includes/indexBottomPanel.html]: @ekamp So I don't have access to the firebase and neither did @itsswift as I recall. I'm not sure we need a firebase for this small bit of data... but if everyone else thinks its necessary I don't mind, it was more of just a quick fix again :p

Comment (itsswift) [inline _includes/indexBottomPanel.html]: You're in charge of the website now friend. Use whatever tech you want.

**Pull request merged**